optimize/client/src/components/Reports/ReportView.js

```diff
@@ -71,7 +71,7 @@ export default function ReportView({report, error, loadReport}) {
   }
 
   const shouldShowCSVDownload = () => {
-    return report?.data?.visualization !== 'number' && report.result?.measures.length === 1;
+    return report?.data?.view?.properties?.includes('rawData') && report.result?.measures.length === 1;
```

Comment:
I think the measures check is not necessary anymore since the raw data is always a single measure report.

Reply:
okay, I didn't know that, I will remove it